Makefile

```diff
@@ -71,7 +87,7 @@ modtidy:
 	go mod tidy
 
 .PHONY: modcheck
-modcheck: modtidy
+modcheck: gen-protobuf modtidy
```

Comment:
I think this is wrong.  Go mod changes should not depend in any way on protobuf (target does not update go.mod). Also for example building zli should not depend on building protoc as it does not have this dependency.

Reply:
That's right